backend/compound_library/src/graphql/compound_types_res.rs

```diff
@@ -0,0 +1,108 @@
+use crate::entities::{compound_instances, compound_types};
+use async_graphql::{ComplexObject, Context, CustomValidator, InputValueError, Object};
+use chrono::Utc;
+use opa_client::subject_authorization;
+use purr::{
+    graph::Builder,
+    read::{read, Trace},
+};
+use sea_orm::{ActiveValue, ColumnTrait, DatabaseConnection, EntityTrait, ModelTrait, QueryFilter};
+use the_paginator::graphql::{CursorInput, ModelConnection};
+
+#[derive(Debug, Clone, Default)]
+pub struct CompoundQuery;
+
+#[derive(Debug, Clone, Default)]
+pub struct CompoundMutation;
+
+#[ComplexObject]
+impl compound_types::Model {
+    async fn compound_instances(
+        &self,
+        ctx: &Context<'_>,
+    ) -> async_graphql::Result<Vec<compound_instances::Model>> {
```

Comment:
`instances` might be a more appropriate name since we're already on a `CompoundType` node